hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1434,6 +1434,15 @@
       run_opts: ["+usb_max_drift=1", "+usb_fast_sof=1"]
       reseed: 1
     }
+    //--------------------------------------------
```

Comment:
could you remove the extra comment lines here? Sorry I know this is nitpicky, but we generally try to adhere to the layout that @sriyerg setup so things have a consistent look and feel.

Reply:
Yes, sure.
The code is not ready yet.
The main purpose of this PR is just to make sure we have the correct understanding of what the test should do.

Reply:
Can you confirm whether the test meets the requirement of the test plan?